plugins/utils/comment-nvim.nix

```diff
@@ -94,6 +96,8 @@ in {
   config = let
     setupOptions = {
       inherit (cfg) padding sticky ignore toggler opleader mappings;
+      pre_hook = cfg.preHook;
+      post_hook = cfg.postHook;
```

Comment:
you need `pre_hook = helpers.mkRaw cfg.preHook` to make it as lua code (same for the other line)

Reply:
Thank you, this worked!